uwblueprint/project-read-backend
# Create enrolment route

### Notion ticket link



[Create an enrolment when a family is registered](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=622fff360dff47bdb0e69707814026dc)



### Implementation description

- !! to be run with the create-enrolment [frontend branch](https://github.com/uwblueprint/project-read-frontend/tree/create-enrolment) !!
- added serializer for creating an enrolment using a family, preferred_class, and session
- updated preferred_class in Enrolment model to be optional (blank=True)



### Steps to test

1. ./manage.py test
2. use the form from the frontend branch to create an enrolment and see if data shows up in Django admin



### What should reviewers focus on?

- robust Python code, correct use of serializers

### Checklist
